uxarray/helpers.py

```diff
@@ -598,6 +601,6 @@ def _replace_fill_values(grid_var, original_fill, new_fill, new_dtype=None):
                          f' dtype: {grid_var.dtype}')
```

Comment:
If it's possible that the `new_fill` is also a `nan`? I am just thinking about some extremely marginal case in the future. But since we have already settled down the `fill_value` to be `INT_MAX` it should be fine now.

Reply:
It actually is possible and likely (see #248), since we will use `nan` as a fill value for floating point arrays.